### PR TITLE
Fix Issue #2750 commandName variable when saving commands

### PR DESCRIPTION
**Pull request by Psychoboy**
Updated to have variable commandName to fix the issue when attempting to save commands via the Web Panels.

### CODE REVIEW

**Comment by Psychoboy** [comment]
Fixes #2750

File: docs/panel-stable/js/pages/commands/default.js [hunk 1]
```diff
@@ -176,7 +176,8 @@ $(function() {
                                     'If checked, the command cannot be used in chat.'))
                                 // Callback function to be called once we hit the save button on the modal.
                         })), function() {
-                            let commandPermission = $('#command-permission'),
+                            let commandName = $('#command-name'),
+                                commandPermission = $('#command-permission'),
```
Comment (gmt2001) [inline docs/panel-stable/js/pages/commands/default.js]: Please do not edit files in the `docs/panel-stable/` folder. This folder is updated by automation upon release

File: docs/panel/js/pages/commands/default.js [hunk 2]
```diff
@@ -176,7 +176,8 @@ $(function() {
                                     'If checked, the command cannot be used in chat.'))
                                 // Callback function to be called once we hit the save button on the modal.
                         })), function() {
-                            let commandPermission = $('#command-permission'),
+                            let commandName = $('#command-name'),
+                                commandPermission = $('#command-permission'),
```
Comment (gmt2001) [inline docs/panel/js/pages/commands/default.js]: Please do not edit files in the `docs/panel/` folder. This folder is updated by automation upon merge

**Review by gmt2001 — CHANGES_REQUESTED** [review]

File: docs/panel-stable/js/pages/commands/default.js (hunk 1 above)
Comment (Psychoboy) [reply docs/panel-stable/js/pages/commands/default.js]: Ah that makes more sense

**Comment by Psychoboy** [comment]
Removed the automated file changes

**Review by gmt2001 — APPROVED** [review]

**Pull request merged**